app.rb

```diff
@@ -0,0 +1,97 @@
+require_relative './lib/item_repository'
+require_relative './lib/order_repository'
+
+
+class Application
```

Comment:
This app.rb is quite long - could be worth separating this so that app.rb only has the 'if __FILE__ == $0' section (lines 89 to 97). And then application.rb can be in the lib folder